src/mca/rmaps/base/rmaps_base_map_job.c

```diff
@@ -448,12 +450,16 @@ void prte_rmaps_base_map_job(int fd, short args, void *cbdata)
                     PRTE_SET_MAPPING_POLICY(jdata->map->mapping, PRTE_MAPPING_BYCORE);
                 }
             } else {
-                /* if package is available, map by that */
-                if (NULL != hwloc_get_obj_by_type(prte_hwloc_topology, HWLOC_OBJ_PACKAGE, 0)) {
+                /* if NUMA is available, map by that */
+                if (NULL != hwloc_get_obj_by_type(prte_hwloc_topology, HWLOC_OBJ_NODE, 0)) {
                     prte_output_verbose(5, prte_rmaps_base_framework.framework_output,
-                                        "mca:rmaps[%d] mapping not set by user - using bypackage",
-                                        __LINE__);
-                    PRTE_SET_MAPPING_POLICY(jdata->map->mapping, PRTE_MAPPING_BYPACKAGE);
+                                        "mca:rmaps[%d] mapping not set by user - using bynuma", __LINE__);
+                    PRTE_SET_MAPPING_POLICY(jdata->map->mapping, PRTE_MAPPING_BYNUMA);
+                } else if (NULL != hwloc_get_obj_by_type(prte_hwloc_topology, HWLOC_OBJ_PACKAGE, 0)) {
+                    /* if package is available, map by that */
+                    prte_output_verbose(5, prte_rmaps_base_framework.framework_output,
+                                        "mca:rmaps[%d] mapping not set by user - using bypackage", __LINE__);
+                    PRTE_SET_MAPPING_POLICY(jdata->map->mapping, PRTE_MAPPING_BYNUMA);
```

Comment:
Looks like a copy/paste typo?
```suggestion
                    PRTE_SET_MAPPING_POLICY(jdata->map->mapping, PRTE_MAPPING_BYPACKAGE);
```